src/util.js

```diff
@@ -22,14 +22,22 @@ const cid = (dagNode, callback) => {
   const firstHash = sha256().update(dagNode.toBuffer(true)).digest()
   const headerHash = sha256().update(Buffer.from(firstHash)).digest()
 
-  const multihash = multihashes.encode(Buffer.from(headerHash), 'dbl-sha2-256')
-  const cidVersion = 1
-  const cid = new CID(cidVersion, 'bitcoin-block', multihash)
+  const cid = hashToCid(Buffer.from(headerHash))
   const err = null
   callback(err, cid)
 }
 
+const hashToCid = (hash) => {
+  const multihash = multihashes.encode(hash, 'dbl-sha2-256')
+  const cidVersion = 1
+  const cid = new CID(cidVersion, 'bitcoin-block', multihash)
+  return cid
+}
+
 module.exports = {
+  hashToCid: hashToCid,
```

Comment:
Needs to be documented

Reply:
Any plan to get this documented in the README as well?

Reply:
That will only get API docs. It's not really public API, it just leaked into it because of the way JS works.